src/workloads/docs/CrudActor.yml

```diff
@@ -46,3 +46,33 @@ Actors:
         Options:
           WriteConcern:
             Level: majority
+  - Repeat: 10
+    Threads: 8
+    # Generate names from Collection0 to Collection<CollectionCount -1>.
+    # The collection names are generated from the current actor id mod CollectionCount. So
+    # for at least one thread per collection the number of Threads must be greater than or equal to CollectionCount.
+    # If Threads < CollectionCount then the names generated will be based on the ActorIds so not all
+    # Collections will be used and it is difficult to predict which collection names will be generated. In this case,
+    # you may want to stick with Collection and explicitly pick the collection that you want.
```

Comment:
This warning reads a bit more ominous than it is.

Would it instead make sense to get the number from a distribution rather than mod count? Functionaly not that different I don't think but perhaps a bit easier to reason about / explain / honest about hard-to-predict. - "Each instance gets CollectionX where X \in {1,n] for n=NumCollections".

Reply:
my concern with a distribution was that it may not give you what you want for low numbers of threads (in one of my tests the prng produced 0,0,0, 1 ... which isn't what I wanted.

I'll lighten the text to make it informational rather than a dire warning.